File: src/command/java/com/my/CommandManager.java
package src.command.java.com.my;

import src.command.java.com.my.command.LikePostCommand;
import src.command.java.com.my.command.SendMessageCommand;

public class CommandManager {
    public static void executeCommand(String command) {
        switch (command.toLowerCase()) {
            case "like post" -> new LikePostCommand().execute();
            case "send message" -> new SendMessageCommand().execute();
        }
    }
}

File: src/command/python/CommandManager.py
from src.command.python.command.LikePostCommand import LikePostCommand
from src.command.python.command.SendMessageCommand import SendMessageCommand


class CommandManager:
    @classmethod
    def executeCommand(cls, command: str):
        match command:
            case "like post": LikePostCommand().execute()
            case "send message": SendMessageCommand().execute()
